docs/openapi.yaml

```diff
@@ -205,6 +205,73 @@ paths:
               examples:
                 error_103_example:
                   $ref: "#/components/examples/error_103_example"
+    put:
```

Comment:
có đổi mật khẩu không nhỉ? hay chia ra thêm 1 endpoint nữa

Reply:
dạ e nghĩ là cũng nên cần nhưng mà mọi người có kịp tiến độ không á a

Reply:
nếu thêm thì thêm 1 API á a, chơ bỏ password nơi thông tin chỉnh sửa k an toàn lắm á a

Reply:
tại sao không an toàn á em

Reply:
e nghĩ là rứa ví e thấy mấy ứng dụng hay web khác họ thường để chỉnh sửa password ở một chỗ khác chứ ko để ở phần chỉnh sửa thông tin cá nhân á a, nên chắc cũng có vấn đề về bảo mật hay gì đó mới như rứa